tox.ini

```diff
@@ -4,38 +4,48 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py34, py35, py36, py37
+envlist = py27, py35, py36, py37, {py35, py36, py37}-async
 
 [testenv]
 commands = coverage erase
-           coverage run -m unittest discover -v []
-           coverage report
+           coverage run -m unittest discover -p "test_*.py" -v []
+           coverage report --ignore-errors
+async_commands = coverage run -m unittest discover -p "async_test_*.py" -v []
```

Comment:
Coverage is gathered but not used. To get full coverage easy way there should be single run over all needed files. Manual report run of last env pass shown 82% on `sendgrid.async_sendgrid`.

Reply:
Fixed for async envs.